### ISSUE TITLE
Missing Velocity.js after upgrading to Ember 1.10.0

### ISSUE THREAD

**Issue by beerlington** [quandl/ember-velocity-mixin#6]
I'm trying to upgrade my application from Ember 1.9.1 to 1.10.0 and am getting the following error when running my test suite or using the application:

```
Assertion Failed: Velocity.js must be installed to use this mixin.
```

I created a [sample app](https://github.com/beerlington/ember-velocity-dependency-issue) that has [instructions in the README](https://github.com/beerlington/ember-velocity-dependency-issue#steps-to-reproduce) to reproduce this issue. It is a barebones ember-cli app (1.11) with a single component generated using `ember g component`.

**Comment by taras** [comment]
@beerlington I just published 0.2.1 that includes fix. Also, this repo is moving to [embersherpa/ember-velocity-mixin](embersherpa/ember-velocity-mixin) see you there :)

**Comment by beerlington** [comment]
Awesome, thanks! The link above to the new repo is busted so here's the right one if anyone stumbles across this in the future https://github.com/embersherpa/ember-velocity-mixin
